programs/ziti-edge-tunnel/instance-config.c

```diff
@@ -169,9 +169,11 @@ bool save_tunnel_status_to_file() {
 }
 
 void cleanup_instance_config() {
-    ZITI_LOG(TRACE, "Backing up current tunnel status");
+    ZITI_LOG(DEBUG,"Backing up current tunnel status ");
     save_tunnel_status_to_file();
+    ZITI_LOG(DEBUG,"save_tunnel_status_to_file done ");
     if (sem_initialized == 0) {
-        uv_sem_destroy(&sem);
+        //uv_sem_destroy(&sem);
```

Comment:
this whole block around &sem seemed unnecessary? I don't know why it's there. can i remove it all? @mary-dcouto